src/ILVerification/src/AccessVerificationHelpers.cs

```diff
@@ -51,7 +51,7 @@ internal static bool CanAccess(this TypeDesc currentClass, TypeDesc targetClass)
                 if ((targetTypeDef.Attributes & TypeAttributes.Public) != 0)
                     return true;
                 else
-                    return currentTypeDef.Module == targetTypeDef.Module || targetTypeDef.Module.GrantsFriendAccessTo(currentTypeDef.Module);
+                    return targetTypeDef.Module.GrantsFriendAccessTo(currentTypeDef.Module);
```

Comment:
I'm not sure folding the `currentTypeDef.Module == targetTypeDef.Module` check into `GrantsFriendAccessTo` is good from readability perspective. I wouldn't expect a method with such name to do that part of the check.